### ISSUE TITLE
Audio shortcuts

### ISSUE THREAD

**Issue by nnrs1** [Genymobile/scrcpy#3798]
I,
Can we switch between audio output on a PC and on the phone using a shortcut without having to restart the terminal again with the "--no-audio" command?
I didn't see this shortcut in this 2.0 update.
Thx

**Comment by rom1v** [comment]
No, and it's not trivial to enable/disable audio forwarding dynamically (recording is impacted, it must know in advance if there is an audio stream because it must write the header before any data packet, etc.)

**Comment by Dixiegamer** [comment]
como puedo desactivar el bluetooth cuando tengo conectado el celular a la pc
ejemplo 
escucho musica en spotify con un dispositivo bluetooth  y al abrir la aplicacion se cancela el audio por todo el audio ahora lo trasmite la pc y se deja de escuchar el audio reproducido

**Comment by rom1v** [comment]
@Dixiegamer Run `scrcpy --no-audio` (if I understand correctly, please write in English)
